examples/next/pages/ui/components/authenticator/sign-in-federated/index.page.tsx

```diff
@@ -10,6 +10,8 @@ Amplify.configure({
   },
 });
 
+I18n.setLanguage('ja');
```

Comment:
FYI, I found this:
> https://github.com/cypress-io/cypress/issues/7890#issuecomment-824676390

I don't know the best way of testing this, but we could consider having an `components/authenticator/localization/en-GB.page.tsx`, `index.ja-JP.page.tsx`, etc. to test some of this behavior.

Not that we do it here, but it's probably something we'll want to document and support via a `.feature`.

Reply:
Yeah, I'm changing this back to english. But I'll take a look at adding some tests in in the next PR once this one get approved. The next PR will have me going through all of the React files and adding translations.